<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snimokCam</class>
 <widget class="QDialog" name="snimokCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-left: 5px;
background-color: rgb(248, 244, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>761</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
border-left: 5px;
background-color: rgb(230, 233, 250);
border: 0.5px solid rgb(101, 126, 253);
border-radius: 8px;
color: rgb(105, 127, 254);
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(234, 237, 253);
border: 0.5px solid rgb(101, 126, 253);
border-radius: 8px;
color: rgb(105, 127, 254);
text-align: center;
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>550</y>
     <width>311</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Зарегестрировать
 человека</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Изменить 
информацию</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>70</y>
     <width>311</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
border-left: 5px;
background-color: rgb(230, 233, 250);
border: 0.5px solid rgb(101, 126, 253);
border-radius: 8px;
color: rgb(105, 127, 254);
font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Люди на фото</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Зарегестрированные люди</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>Незарегистрированные люди</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
